resolution/design/00_resolution.md

```diff
@@ -0,0 +1,331 @@
+# PACE Instrumental Resolution Convolution
+**2020-04-06**
+
+# Overview
+
+Inelastic neutron scattering counts neutron absorption events as a function of
+momentum, ![\mathbf{Q}], and energy, ![E], transferred from the neutron to the sample.
+The *rate* at which events are detected is proportional to the so-called doubly-differential scattering cross section
+
+![S(\mathbf{Q},E) \propto \frac{\partial^2\sigma}{\partial \mathbf{Q} \partial E}]
+
+which is directly comparable to theoretical models for excitation processes in a condensed matter system.
+
+The momentum transferred to the sample is the difference of the initial and final momenta of the neutron
+
+![\mathbf{Q} = \mathbf{k}_\text{i} - \mathbf{k}_\text{f}]
+
+and the energy transferred is the difference of the initial and final kinetic energy of the neutron
+
+![E = \frac{\hbar^2}{2m_\text{n}}(k_\text{i}^2 - k_\text{f}^2)]
+
+Neutron scattering instruments are typically constructed to allow an appreciable uncertainty in
+the initial and final momenta of detected neutrons, ![\delta\mathbf{k}_\text{i}] and ![\delta\mathbf{k}_\text{f}], respectively.
+These uncertainties naturally give rise to uncertainties in the momentum, ![\delta\mathbf{Q}], and energy, ![\delta E], transferred to the sample;
+and therefore uncertainties in the doubly-differential cross section.
+In the interest of simplified notation we can define a momentum-energy four vector ![\mathbb{Q} \equiv (\mathb{Q},E)], with uncertainty ![\delta\mathbb{Q}].
+The uncertainty in momentum-energy at a given ideal momentum-energy point gives rise to the instrumental resolution function ![R(\mathbb{X};\mathbb{Q})].
```

Comment:
I'll extend this part of the document and introduce the covariance matrix, V<sub>ij</sub>(**Q**,E), since this is what is used extensively to approximate the resolution of, e.g., triple-axis neutron spectrometers (c.f, [ResLib](https://neutron.ethz.ch/research/resources/reslib.html)).

The notation I'm using here is probably not ideal, but the idea that I'm attempting to convey is that the resolution *function* is defined for a specific instrument configuration [which is characterised by a single (**Q**,E) point] and is valid over all momentum-energy space.
The idea in the notation, as with model functions S(**Q**,E; **p**), that that properties which *define* the continuous function are listed following a semicolon.
The resolution function at a specific point, say (**Q**<sub>0</sub>,E<sub>0</sub>), is defined by the instrument configuration and is continuous over all (**Q**,E) &mdash; so, R(**Q**,E; **Q**<sub>0</sub>,E<sub>0</sub>).
We could stick with exactly the same symbols for defining the covariance, but then would specify V<sub>ij</sub>(**Q**<sub>0</sub>,E<sub>0</sub>), when my goal was to emphasize that it too is continuous in the experimentally accessible part of (**Q**,E) space.

As for why this document does not use R(**Q** - **Q**<sub>0</sub>, E - E<sub>0</sub>;  **Q**<sub>0</sub>,E<sub>0</sub>): Such a construction is useful when you define the instrumental resolution function as a hyperellipsoid as it simplifies the 4-D Gaussian expression, but as such a definition is beyond the scope of this document (and isn't necessary for all resolution models) I've decided against introducing an origin shift in the integrals.

Reply:
Thanks for the clarification.